HadesScheduler/docker/docker.go

```diff
@@ -299,5 +282,20 @@ func (s DockerStep) execute(ctx context.Context) error {
 	}
 
 	s.logger.Debug("Container completed", slog.Any("container_id", resp.ID), slog.Any("image", s.Image))
+
+	// Write the container logs to NATS
+	err = processContainerLogs(ctx, s.cli, s.nc, resp.ID, job_id)
```

Comment:
Is there a way to publish the logs live? I think now that we have the chance to implement the foundations for live logs - we should do it :)